src/con_utils.ts

```diff
@@ -124,6 +137,41 @@ function parseConnectDsnAndArgs({
     );
     serverSettings = server_settings;
   }
+
+  if (
+    !(
+      dsn ||
+      host ||
+      port ||
+      process.env.EDGEDB_HOST ||
+      process.env.EDGEDB_PORT
+    )
+  ) {
+    if (process.env.EDGEDB_INSTANCE) {
+      dsn = process.env.EDGEDB_INSTANCE;
+    } else {
+      const dir = process.cwd();
+      if (!fs.existsSync(path.join(dir, "edgedb.toml"))) {
+        throw new errors.ClientConnectionError(
+          "no `edgedb.toml` found and no connection options specified"
```

Comment:
```suggestion
          "no `edgedb.toml` found and no connection options specified" +
            " neither via arguments to connect API nor via environment" +
            " variables EDGEDB_HOST/EDGEDB_PORT or EDGEDB_INSTANCE"
```

Reply:
I think `neither` should be `either` and `nor` changed to `or`